test/src/test/java/hudson/model/DirectoryBrowserSupportTest.java

```diff
@@ -685,7 +685,7 @@ public void symlink_outsideWorkspace_areNotAllowed() throws Exception {
             assertThat(zipPage.getWebResponse().getStatusCode(), equalTo(HttpURLConnection.HTTP_OK));
 
             List<String> entryNames = getListOfEntriesInDownloadedZip((UnexpectedPage) zipPage);
-            assertThat(entryNames, containsInAnyOrder(
+            assertThat(entryNames, hasItems(
```

Comment:
As in commit 14cf0bfcbc98fd10f5680bea244385c4f2fb5139, please strengthen the assertion by maintaining the original `containsInAnyOrder` and adding any newly expected entries to the test.

Reply:
Will do today!

Reply:
@basil my latest commit should've fixed that. Could you double check?